libraries/lib-uuid/Uuid.h

```diff
@@ -17,18 +17,43 @@
 namespace audacity
 {
 
+/*!
+Utility class that generates and parses UUIDs.
```

Comment:
@brief
should start that line.  Or break it out before the `/*!` on a `//!` line.